lib/ueberauth/strategy/auth0/oauth.ex

```diff
@@ -14,8 +14,10 @@ defmodule Ueberauth.Strategy.Auth0.OAuth do
         client_secret: {:system, "AUTH0_CLIENT_SECRET"}
   """
   use OAuth2.Strategy
+  alias OAuth2.Client
+  alias OAuth2.Strategy.AuthCode
 
-  def options() do
+  def options do
```

Comment:
You don't have to if you think `def options do` is the better way. I just voiced my opinion.